lib/fastlane/plugin/firebase_app_distribution/actions/firebase_app_distribution_action.rb

```diff
@@ -191,6 +166,47 @@ def self.is_firebasecmd_supported?(cmd)
 
         true
       end
+
+      def self.get_token
+        client = Signet::OAuth2::Client.new(
+          token_credential_uri: 'https://oauth2.googleapis.com/token',
+          client_id: FirebaseAppDistributionLoginAction::CLIENT_ID,
+          client_secret: FirebaseAppDistributionLoginAction::CLIENT_SECRET,
+          refresh_token: ENV["FIREBASE_TOKEN"]
+        )
+
+        client.fetch_access_token!
+        return client.access_token
+      rescue Signet::AuthorizationError => error
+        UI.crash!("Wrong value for FIREBASE_TOKEN")
+      end
+
+      def self.get_app(params)
```

Comment:
Consider renaming to `validate_app!` since we aren't actually returning a value but just validating the app exists and has a contact email.